tests/stein_tests/constants.py

```diff
@@ -0,0 +1,20 @@
+# Debug Flags

```

Comment:
will a separate folder created for legacy tests? can we reuse constants, testutils with legacy tests?